Documentation/learning/api_guarantees.md

```diff
@@ -41,12 +41,16 @@ As with all distributed systems, it is impossible for etcd to ensure [strict con
 
 #### Isolation
 
-etcd ensures [serializable isolation][serializable_isolation], which is the highest isolation level available in distributed systems. Read operations will never observe any intermediate data.
+etcd ensures [serializable isolation][serializable_isolation], which is the highest isolation level available in transactional database management systems. Read operations will never observe any intermediate data.
```

Comment:
i think what @aphyr suggest is to simply merge isolation section with consistency section. then we can claim that etcd supports strict serializability.